test/connections.ts

```diff
@@ -395,3 +395,11 @@ describe('link lookup and iteration', function () {
         });
     });
 });
+
+describe('container', function () {
+    it('create_connection should return an instance of connection', function (done) {
+        var connection: rhea.Connection = rhea.create_connection();
+        assert.equal(connection.constructor.name, 'Connection');
```

Comment:
This test seems to be failing under node 4. Perhaps a better test anyway would be to ensure that you can connect the returned object successfully?